digitalocean/app_spec.go

```diff
@@ -221,6 +251,14 @@ func appSpecComponentBase() map[string]*schema.Schema {
 				Schema: appSpecGitLabSourceSchema(),
 			},
 		},
+		"image": {
```

Comment:
Can we move this out of `appSpecComponentBase()` and into both `appSpecServicesSchema()` and `appSpecWorkerSchema()`? It is for elements that are shared between all component types, and static sites do not user Docker images.

Reply:
ok, I see